ykrt/src/mt.rs

```diff
@@ -237,7 +237,11 @@ impl MT {
                     }
                     match loc.lock() {
                         Ok(x) => ls = x,
-                        Err(()) => unreachable!(),
+                        Err(()) => {
```

Comment:
What prompted this change? Did you encounter a crash? If so, do we have a test?

Reply:
When reading the code, I realised this is reachable. Testing this sort of nondeterministic code exhaustively is impossible, unfortunately.

Reply:
ok